src/com/rcv/Tabulator.java

```diff
@@ -675,13 +680,62 @@ private boolean hasContinuingCandidates(SortedMap<Integer, Set<String>> rankToCa
       } // end looping over the rankings within one ballot
     } // end looping over all ballots
 
+    // Take the tallies for this round for each precinct and merge them into the main map tracking
+    // the tallies by precinct.
+    for (String precinct : roundTallyByPrecinct.keySet()) {
+      // the set of round tallies that we've built up so far for this precinct
+      Map<Integer, Map<String, BigDecimal>> roundTalliesForPrecinct =
+          precinctRoundTallies.get(precinct);
+      roundTalliesForPrecinct.put(currentRound, roundTallyByPrecinct.get(precinct));
+    }
+
     return roundTally;
   }
 
+  // function: getNewTally
+  // purpose: create a new initialized tally with all continuing candidates
```

Comment:
Needs params documented.

Reply:
Oops. Thanks for flagging (here and below).